versioned_docs/version-6.x/nesting-navigators.md

```diff
@@ -111,27 +115,30 @@ In your app, you will probably use these patterns depending on the behavior you
 
 Consider the following example:
 
+<samp id="nested-navigator-screen" />
+
 ```js
 function Root() {
   return (
-    <Drawer.Navigator>
-      <Drawer.Screen name="Home" component={Home} />
-      <Drawer.Screen name="Profile" component={Profile} />
-      <Stack.Screen name="Settings" component={Settings} />
```

Comment:
I think we shouldn't remove the `component` props in here. It has a semantic meaning for the reader and in this example highlighting that both navigators and screens can be passed to this prop.

I understand that in a context of `snack` these screens are kind of boilerplate but for didactic purposes it's not a bad thing.